### PR TITLE
more boilerplate, some three stuffs, a few missing deps

### PR DESCRIPTION
**Pull request by apathetic**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1 +1,22 @@
...
+
+import App from "./App";
+import Store from "./store/";
+
+// import "./index.css";
+// import registerServiceWorker from "./registerServiceWorker";
+
+// Let the reducers handle initial state
+const initialState = {};
+const store = Store(initialState);
```
Comment (raedatoui) [inline src/index.js]: use the Redux createStore https://github.com/reactjs/redux/blob/master/docs/api/createStore.md

File: src/store/index.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { createStore } from "redux";
+import reducers from "./reducers";
+
+export default function Store(initialState = {}) {
```
Comment (raedatoui) [inline src/store/index.js]: no need to encapsulate the store in a class, a function would be nice
`export const makeStore = () => createStore(reducer, defaultState, enhancer);`

Comment (apathetic) [reply src/store/index.js]: oh yeah; there was also some middleware stuffs in here before which i stripped out, not sure if we'll need eventually or not

Comment (raedatoui) [reply src/store/index.js]: i thought it was a class, but its a function. i guess stylistically function names are not capitalized

Comment (apathetic) [reply src/store/index.js]: yeah react is weird; all its functions seem to be capitalized. it's stupid

**Pull request merged**